src/metatensor/models/cli/train.py

```diff
@@ -411,7 +411,17 @@ def _train_model_hydra(options: DictConfig) -> None:
             extra_log_message = f" with index {i}"
 
         logger.info(f"Evaluating training dataset{extra_log_message}")
-        _eval_targets(exported_model, train_dataset)
+        if len(train_dataset) == 0:
+            eval_options = {}
```

Comment:
How can this happen? If there is no data in the training dataset this should fail during training. And if yes please a [test](https://app.codecov.io/gh/lab-cosmo/metatensor-models/pull/150/blob/src/metatensor/models/cli/train.py#L415). I can not really see that the lines below do the correct thing if they get an empty dict

Reply:
Ok, removed

Reply:
It's gone in the "apply suggestions" commit